src/spikeinterface/sortingcomponents/tools.py

```diff
@@ -19,6 +19,9 @@ def make_multi_method_doc(methods, ident="    "):
 
 
 def get_prototype_spike(recording, peaks, job_kwargs, nb_peaks=1000, ms_before=0.5, ms_after=0.5):
+    # TODO for Pierre: this function is really unefficient because it runa full pipeline only for a few
+    # spikes, which leans that traces are entirally computed!!!!!
+    # Please find a better way
```

Comment:
```suggestion
    # TODO for Pierre: this function is really inefficient because it runs a full pipeline only for a few
    # spikes, which means that all traces need to be accesses! Please find a better way
```